.eslintrc

```diff
@@ -48,6 +48,10 @@
     "no-throw-literal": 2,
     "no-trailing-spaces": 2,
     "no-unused-expressions": 0,
+    "no-unused-vars": [2, {
+      "argsIgnorePattern": "^var_|opt_|unused_|_",
```

Comment:
actually sorry, the only prefix it skips (the google camelcase plugin) is `opt_` but that changes what we're trying to convey with the param... (since these unused vars aren't necessary optional)